community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/participant/StoreMigrator.java

```diff
@@ -634,6 +714,30 @@ public void moveMigratedFiles( File migrationDir, File storeDir, String versionT
                 true, // allow to skip non existent source files
                 ExistingTargetStrategy.OVERWRITE, // allow to overwrite target files
                 StoreFileType.values() );
+        // Since some of the files might only be accessible through the page cache (i.e. block devices), we also try to
+        // move the files with the page cache.
```

Comment:
I like this pattern of first trying to finish the job with `FileSystemAbstraction` and then "clean up" what's left with `PageCache`.